draft-ietf-httpbis-resumable-upload.md

```diff
@@ -383,30 +382,18 @@ upload-draft-interop-version: 2
 
 The Upload Creation Procedure ({{upload-creation}}) supports arbitrary methods including `PATCH`, therefore it is not possible to identify the procedure of a request purely by its method. The following algorithm is RECOMMENDED to identify the procedure from a request for a generic implementation:
```

Comment:
I think we could get rid of this entire section. It's no longer ambiguous what the procedure is since the server determines the URL for appending, retrieving and cancelling.

Reply:
I get where you are coming from and the relevancy of this section has decreased with this PR, yes. I do think that this section is helpful for people looking to start a server-side implementation. However, I am not sure if such content is best placed in the draft itself.